### DISCUSSION TITLE
upload failed

### DISCUSSION THREAD

**Discussion by chinlumg** [mediacms-io/mediacms#397] [General]
<img width="1014" alt="截圖 2022-01-22 22 15 57" src="https://user-images.githubusercontent.com/12997481/150641975-8c97dce8-d062-4692-bab7-7e486b7cbe39.png">

**Comment by chinlumg** [comment]
<img width="1013" alt="截圖 2022-01-22 22 17 50" src="https://user-images.githubusercontent.com/12997481/150642074-5e4218a0-44be-431b-b85c-785ca21b020b.png">

**Reply by chinlumg** [reply]
the issue that I have is that any file goes into retrying mode instantly, some cases it pushes X% of the file, say 5%? then it says retrying 1/2 ... 2/2 upload failed. (use Ubuntu 20  Server Installation)

**Comment by srf-1** [comment]
I'm having the same issue (docker installation on windows 10).

Happens when both uploading video's and profile images.

Any ideas on what's causing it or how to resolve?

The only thing I've found in the logs so far is

2023-06-03 08:37:13 2023/06/02 22:37:13 [warn] 56#56: *254 a client request body is buffered to a temporary file /var/lib/nginx/body/0000000047, client: 172.20.0.1, server: , request: "POST /user/admin/edit HTTP/1.1", host: "localhost", referrer: "http://localhost/user/admin/edit"

**Reply by mgogoulos** [reply]
That could be related or not, specially if it's Docker on Windows. The mediacms process need to be able to write on media_files/ to upload the files, probably you check the permissions on this folder, not sure whether `tail -f logs` can bring something more relevant too

**Reply by srf-1** [reply]
thanks, I'll give that a go
